### ISSUE TITLE
Prevent using or calling uninstalled packages

### ISSUE THREAD

**Issue by Compro-Prasad** [brotzeit/rustic#22]
Like flycheck. I haven't installed flycheck but this package always calls it which I don't like. If not available either offer to install it or don't try to enable it.

**Comment by brotzeit** [comment]
Yeah, my solution still doesn't work for every case. But I'm a little confused since flycheck shouldn't be installed by rustic. The rustic flycheck file will only be required when flycheck is loaded. And you shouldn't be offered to install it.
Are you absolutely sure you mean flycheck ?
If so please try if you get the same issue with a minimal config and describe your steps.

Regarding eglot and lsp-mode, I have to fix automatic requiring.

**Comment by Compro-Prasad** [comment]
I mean that flycheck-mode is called after opening any rust file.

**Comment by brotzeit** [comment]
Are you sure that flycheck wasn't installed by some package as dependency ? I will change this, but I can't explain why rustic would call flycheck if it isn't already installed.
